### PR TITLE
 Adding Files of Watch new Movie #80

### PR DESCRIPTION
**Pull request by Zaheer-zk**

### CODE REVIEW

File: watch-new-movie/spec.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  - See New Movie which is sorted by rating and newest.
+
+  - User can also search the movie by there names.
+
+  # API Documentation is available at https://developers.themoviedb.org/3/getting-started/introduction
+
+  - Download the starter code and go through the README.md file.
+
+  The `README.md` file will provide further details about the project. The `style-guide.html` file is where you'll find colors, fonts, etc.
```
Comment (Pranavtechie) [inline watch-new-movie/spec.yml:25]: There are NO readme.md file and style-guide available right, so you can delete this line

File: watch-new-movie/spec.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  # Your users should be able to:
+
+  - See New Movie which is sorted by rating and newest.
+
+  - User can also search the movie by there names.
+
+  # API Documentation is available at https://developers.themoviedb.org/3/getting-started/introduction
+
+  - Download the starter code and go through the README.md file.
```
Comment (Pranavtechie) [inline watch-new-movie/spec.yml:23]: You can delete this line as well

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
Hey @Zaheer-zk, you have done a great job in breakdown this project, 

However, I can see a lot of css code snippets in the Instructions file, this is not a bad thing, but it would be great if you can also drop some explanation for why the property is used and how this will affect the style of the page. 

These changes would drastically improves the learning experience of the user while building the project.

**Comment by Zaheer-zk** [comment]
Added explanation on  css code snippets in the Instructions file.

**Comment by Pranavtechie** [comment]
these commits are merged from https://github.com/codedamn/projects/pull/229

**Pull request closed without merging**